src/components/Calculator.css

```diff
@@ -0,0 +1,134 @@
+li {
+  list-style: none;
+}
+
+ul {
+  width: 100%;
+  height: 100%;
+  flex-wrap: wrap;
+
+  /* justify-content: flex-end; */
+}
+
+button {
+  width: 100%;
+  height: 100%;
+  background-color: transparent;
+}
+
+button:hover {
+  cursor: pointer;
+  background-color: greenyellow;
+}
+
+.d-flex {
+  display: flex;
+}
+
+.center {
+  justify-content: center;
+  align-items: center;
+}
+
+.calc-result {
+  width: 100%;
+  background-color: grey;
+  color: white;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 1rem;
+}
+
+.calc-item {
+  width: 25%;
+}
+
+.calc-zero {
+  justify-self: start;
+  width: 50%;
+}
+
+.o-0 {
+  order: 0;
+}
+
+.o-1 {
+  order: 14;
+}
+
+.o-2 {
+  order: 15;
+}
+
+.o-3 {
+  order: 16;
+}
+
+.o-4 {
+  order: 10;
+}
+
+.o-5 {
+  order: 11;
+}
+
+.o-6 {
+  order: 12;
+}
+
+.o-7 {
+  order: 5;
+}
+
+.o-8 {
+  order: 6;
+}
+
+.o-9 {
+  order: 7;
+}
+
+.o-10 {
+  order: 1;
+}
+
+.o-11 {
+  order: 2;
+}
+
+.o-12 {
+  order: 3;
+}
+
+.o-13 {
+  order: 4;
+  background-color: tomato;
```

Comment:
- [ ] OPTIONAL: I recommend you to stick with the original background color. 
![image](https://user-images.githubusercontent.com/64894927/129937190-f25f5aa7-7c40-4fdc-99e6-54e83563806c.png)
![image](https://user-images.githubusercontent.com/64894927/129937237-41f9ebe0-5f9e-4a23-93b1-7622aa7eb75c.png)